src/mikro-orm.config.ts.example

```diff
@@ -7,10 +7,13 @@ import { SeedManager } from '@mikro-orm/seeder';
 
 export default defineConfig({
   host: 'localhost',
-  port: 3307,
+  port: 3306,
```

Comment:
3307 is correct

https://github.com/mikro-orm/nestjs-realworld-example-app/blob/master/docker-compose.yml#L9

Reply:
Sorry, I forgot that this setup is for dicker. I'm using a native MySQL local server